script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -127,37 +127,37 @@ I turned around, facing the voice behind me.
 There was a nurse in front of the door with a disturbed look on her face.
 <Page36>
 "How did you break the bed, Shiki-kun?"
-He kept on asking me "how" I broke the bed, not so much as why I did it.
+He kept on asking me 'how' I broke the bed, not so much as why I did it.
 <Page37>
-"I traced the line and it broke.
+"I traced a line and it broke.
 Hey, why does the hospital have cracks everywhere?"
 <Page38>
-"Okay that's enough, Shiki-kun. There are no such lines. So tell me, how did you break the bed? I won't get angry, so tell me."
+"Okay. That's enough, Shiki-kun. There are no such lines. So tell me, how did you break the bed? I won't get angry, so tell me."
 <Page39>
-"―――I already told you. All I did was trace the lines."
+"―――I already told you. All I did was trace the line."
 "...I see. We'll talk about it tomorrow then."
 <Page40>
 The doctor sighed.
 Ever since I mentioned the lines, that kind smile of his was nowhere to be seen.
-I was sure that "switch" of his had broken.
-Seems like this hospital was full of so many fragile things.
+I was sure that his 'switch' had broken.
+Seemed like this hospital was full of so many fragile things.
 <Page41>
 In the end, no one believed my story.
 <Page42>
 No matter what it was, if I cut its lines with a knife, it got sliced clean.
 I didn't need to put any force into it.
 I could cut it as easily as I could cut paper with scissors.
 <Page43>
-Whether it was the bed, a chair, the walls, or the floor.
-...And even though I had never attempted it, maybe, no, definitely humans too.
+Whether it was a bed, a chair, the walls, or the floor.
+...And even though I had never tried, maybe, no, definitely humans too.
 <Page44>
 It looked like no one else could see these lines.
 Lines that only I could see.
 <Page45>
 They were just like patchwork.
 Right after stitching up a wound.
-Sort of like the mark on my chest that was still healing after surgery. And just like that, could be torn apart with a touch.
-Because otherwise, there was no way a kid could slice through a wall with strength alone.
+Like the mark on my chest that was still healing after surgery. Which could be torn apart with a touch.
```

Comment:
Both versions here seem a bit off to me. 
> アレはきっと、ツギハギなんだ。
> 傷口を縫ったあと。
> 僕の胸みたいに、手術をしてもまだ治りきっていない、触れるだけでボロボロに散らばっていく<印|しるし>だと思う。

Perhaps
> They were just like patchwork.
> The result of stitching up a wound.
> Marks just like the one on my chest, still not healed even after surgery, which feels as though it could shatter into pieces at the lightest though.

Reply:
typo of touch as though.